backend/src/main/java/gov/cdc/usds/simplereport/service/DataHubUploaderService.java

```diff
@@ -95,17 +174,17 @@ private void _uploadCSVDocument(final String apiKey) throws IOException, RestCli
             return execution.execute(request, body);
         })).build();
 
-        _uploadResultId = restTemplate.postForObject(DATAHUB_UPLOAD_URL, contentsAsResource, String.class);
+        _resultJson = restTemplate.postForObject(_config.getUploadUrl(), contentsAsResource, String.class);
     }
 
     public String creatTestCVSForDataHub(String lastEndCreateOn) {
```

Comment:
`creatTestCVSForDataHub` -> `createTestCVSForDataHub` added the missing e at the end of create

Reply:
Test CSV might be better than CVS? 😁